src/rpc.js

```diff
@@ -22,146 +43,151 @@ module.exports = (myId, requestHandler, secure) => {
   let cbs = {}
   let id = 1
 
-  const source = Pushable()
-  const sink = pull.drain(data => {
-    log('got %s %s', data.type, data.id)
+  const source = Pushable() // TODO: since we're in pull.drain, the return does nothing yet. need to be fixed. possibly need an entirely new rpc.
```

Comment:
What should be done here is that instead of using pull streams, the RPC should be handled using promises (thus removing the need for the overly complex `cbs[rid]` system, which was necessary with callbacks)